x-pack/legacy/plugins/canvas/canvas_plugin_src/expression_types/embeddable.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ExpressionType } from '../../../../../../src/plugins/data/common/expressions';
+// @ts-ignore Untyped Local
+import { MAP_SAVED_OBJECT_TYPE } from '../../../maps/common/constants';
```

Comment:
Let's create a single file within Canvas, perhaps `embeddable_types.ts`, that imports and exports these.  It will make the code easier to read, and allow us to add embed types as they arrive.